paymentsheet/src/main/res/values/strings.xml

```diff
@@ -12,5 +12,5 @@
     <string name="title_bank_account" tools:ignore="MissingTranslation">Bank Account</string>
     <string name="address_label_name" tools:ignore="MissingTranslation">Name</string>
     <string name="address_label_country" tools:ignore="MissingTranslation">Country</string>
-    <string name="sofort_mandate">Need to fill in the sofort mandate acceptance.</string>
+    <string name="sofort_mandate">By providing your payment information and confirming this payment, you authorise (A) {businessName} and Stripe, our payment service provider, to send instructions to your bank to debit your account and (B) your bank to debit your account in accordance with those instructions. As part of your rights, you are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited. Your rights are explained in a statement that you can obtain from your bank. You agree to receive notifications for future debits up to 2 days before they occur.</string>
```

Comment:
{businessName} needs to be changed to Android placeholder format. This string already exists in the `payments-core` module and it looks like it's not used, can be deleted there.

Reply:
I have the planned in a future review: MOBILESDK-315